packages/console/src/components/AppContent/index.tsx

```diff
@@ -1,15 +1,22 @@
-import React, { ReactNode, useEffect } from 'react';
+import { useLogto } from '@logto/react';
+import React, { useEffect } from 'react';
+import { Outlet, useHref } from 'react-router-dom';
 
+import Content from './components/Content';
+import Sidebar from './components/Sidebar';
+import Topbar from './components/Topbar';
 import * as styles from './index.module.scss';
 
 type Theme = 'light';
 
 type Props = {
-  children: ReactNode;
   theme: Theme;
 };
 
-const AppContent = ({ children, theme }: Props) => {
+const AppContent = ({ theme }: Props) => {
+  const { isAuthenticated, signIn } = useLogto();
+  const href = useHref('/callback');
```

Comment:
What's good of using `useHref` here, instead of just the `/callback` string?

Reply:
we mount AC under `{domain}/console` thus the callback url will be `{domain}/console/callback`. simply using `/callback` will cause problem.